README.md

```diff
@@ -2048,208 +2071,173 @@ The provider name is the xml element name to use when configuring.
+    </tr>
     <tr>
-      <td><tt>uuid</tt></td>
-      <td><p>Outputs random UUID as field value. Handy when you want to provide unique identifier
-      for log lines
-      </p>
+      <td><tt>tags</tt></td>
+      <td><p>Outputs logback markers as a comma separated list.</p>
         <ul>
-          <li><tt>fieldName</tt> - Output field name (<tt>uuid</tt>)</li>
-          <li><tt>strategy</tt> - UUID generation strategy (<tt>random</tt>). Supported options: <ul><li><tt>random</tt> - for Type 4 UUID</li>
-          <li><tt>time</tt> - for Type 1 time based UUID</li>
-          </ul></li>
-          <li><tt>ethernet</tt> - Only for 'time' strategy. When defined - MAC address to use for location part of UUID. Set it to <tt>interface</tt> value to use real underlying network interface or to specific values like <tt>00:C0:F0:3D:5B:7C</tt></li>          
+          <li><tt>fieldName</tt> - Output field name (<tt>tags</tt>)</li>          
         </ul>
-	      <p>Note: The <a href="https://mvnrepository.com/artifact/com.fasterxml.uuid/java-uuid-generator/"><tt>com.fasterxml.uuid:java-uuid-generator</tt></a> optional dependency must be added to applications that use the `uuid` provider.</p>
       </td>
     </tr>
     <tr>
-      <td><tt>sequence</tt></td>
-      <td>
-        <p>
-          Outputs an incrementing sequence number for every log event.
-          Useful for tracking pottential message loss during transport (eg. UDP)
-        </p>
+      <td><tt>throwableClassName</tt></td>
+      <td><p>(Only if a throwable was logged) Outputs a field that contains the class name of the thrown Throwable.</p>
         <ul>
-          <li><tt>fieldName</tt> - Output field name (<tt>sequence</tt>)</li></ul>
+          <li><tt>fieldName</tt> - Output field name (<tt>throwable_class</tt>)</li>
+          <li><tt>useSimpleClassName</tt> - When true, the throwable's simple class name will be used. When false, the fully qualified class name will be used. (<tt>true</tt>)</li>
+        </ul>
       </td>
     </tr>
+    <tr>
+      <td><tt>throwableRootCauseClassName</tt></td>
+      <td><p>(Only if a throwable was logged) Outputs a field that contains the class name of the root cause of the thrown Throwable.</p>
+        <ul>
+          <li><tt>fieldName</tt> - Output field name (<tt>throwable_root_cause_class</tt>)</li>
+          <li><tt>useSimpleClassName</tt> - When true, the throwable's simple class name will be used. When false, the fully qualified class name will be used. (<tt>true</tt>)</li>
+        </ul>
+      </td>
+    </tr>  
   </tbody>
 </table>
 
 
 
-#### Providers for AccessEvents  
+### Providers for AccessEvents  
 
-The table below lists the available providers for AccessEvents, and their configuration properties (defaults in parentheses).
+The table below lists the available providers for _AccessEvents_, and their configuration properties (defaults in parentheses).
 The provider name is the xml element name to use when configuring.
```

Comment:
Mention and provide a link to the common providers section somewhere in this paragraph.

Perhaps:

```suggestion
The [common providers mentioned above](#providers-common-to-loggingevents-and-accessevents), and the providers listed in the table below, are available for _AccessEvents_.
The provider name is the xml element name to use when configuring.  Each provider's configuration properties are shown, with default configuration values in parenthesis.
```